airbyte-webapp/src/components/base/Tooltip/InfoTooltip.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import { InfoIcon } from "components/icons/InfoIcon";
+
+import styles from "./InfoTooltip.module.scss";
+import { Tooltip } from "./Tooltip";
+
+export const InfoTooltip: React.FC = ({ children }) => {
+  return (
+    <Tooltip
+      control={
+        <div className={styles.container}>
+          <div className={styles.icon}>
```

Comment:
Why have this intermediate div with its own styling that contains `<InfoIcon>`, instead of just putting `className={styles.icon}` on the `<InfoIcon>` component directly? Can those positioning styles not be applied directly to SVGs or something like that?